hwmgr-plugins/metal3/controller/baremetalhost_manager.go

```diff
@@ -47,6 +47,7 @@ const (
 	NodeNameAnnotation             = "clcm.openshift.io/node-name"
 	BmhDeallocationDoneAnnotation  = "clcm.openshift.io/deallocation-complete"
 	BmhErrorTimestampAnnotation    = "clcm.openshift.io/bmh-error-timestamp"
+	GenerateSeedImageAnnotation    = "clcm.openshift.io/generate-seed-image"
```

Comment:
Do we want to consider naming this something more generic in case there are other scenarios where this may be applicable in the future, skip-cleanup or something like that

Reply:
Sure, any suggestions?

Reply:
"skip-cleanup " sounds good